addons/account/tests/test_account_payment_register.py

```diff
@@ -896,7 +896,7 @@ def test_register_payment_multi_currency_rounding_issue_writeoff_higher_amount_r
                 'amount_currency': -0.04,
                 'reconciled': False,
             },
-            # Liquidity line:
+            # Liquidity line
```

Comment:
After adding whitespaces everywhere, now you are removing a poor colon like that! You are so mad! :scream_cat:

Reply:
I want to leave my mark on this class 😎